drivers/hmis/spec/models/hmis/hud/project_spec.rb

```diff
@@ -82,11 +83,22 @@ def selected_instances
       expect(selected_instances.size).to eq(0)
     end
 
```

Comment:
This is pretty niche/intensive-to-QA behavior that involves both the frontend and backend, so I think it's a good candidate for a system test.

For example:
* for user who has access to edit [CLS], open the enrollment dashboard for an enrollment that has [CLS] but for which no [CLS form] is enabled in the project. (Or, enabled for different client type). User should be able to see and edit old [CLS], but not add a new one.
  * (and conversely if no CLS data exists, they should not see the tab)
* for user who has access to edit [CLS], open the project dashboard for a project that has any [CLS] but for which no [CLS form] is enabled in the project. User should be able to see the [CLS] tab.
  * (and conversely if no CLS data exists, they should not see the tab)

Reply:
Whew, thank you for suggesting this! System test was a great idea, it turned up a couple of interesting bugs, some of which I'd like to reason through together - I'll add some more inline comments, and we can discuss tomorrow.